src/Svelte/doc/index.rst

```diff
@@ -0,0 +1,130 @@
+Symfony UX Svelte
+=================
+
+Symfony UX Svelte is a Symfony bundle integrating `Svelte`_ in
+Symfony applications. It is part of `the Symfony UX initiative`_.
+
+Svelte is a JavaScript framework for building user interfaces.
+Symfony UX Svelte provides tools to render Svelte components from Twig,
+handling rendering and data transfers.
+
+Symfony UX Svelte supports Svelte 3 only.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then install the bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-svelte
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+You also need to add the following lines at the end to your ``assets/app.js`` file:
+
+.. code-block:: javascript
+
+    // assets/app.js
+    import { registerSvelteControllerComponents } from '@symfony/ux-svelte';
+
+    // Registers Svelte controller components to allow loading them from Twig
+    //
+    // Svelte controller components are components that are meant to be rendered
+    // from Twig. These component can then rely on other components that won't be
+    // called directly from Twig.
+    //
+    // By putting only controller components in `svelte/controllers`, you ensure that
+    // internal components won't be automatically included in your JS built file if
+    // they are not necessary.
+    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\.svelte$/));
+
+To make sure Svelte components can be loaded by Webpack Encore, you need to add and configure
+the `svelte-loader`_ library in your project :
+
+.. code-block:: terminal
+
+    $ npm install svelte-loader@^3.0.0 --save-dev
+
+    # or use yarn
+    $ yarn add svelte-loader@^3.0.0 --dev
```

Comment:
Let's remove the version from both of these commands.

Also, add `svelte` itself, as that will also be needed.